### PR TITLE
Update Dataproc python walkthrough and submit_job_to_cluster.py code per latest Dataproc Python quickstart.

### PR DESCRIPTION
**Pull request by aman-ebay**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-dataproc/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/snippets/submit_job_to_cluster.py [hunk 1]
```diff
@@ -1,322 +1,138 @@
...
+    operation = job_client.submit_job_as_operation(
+        request={"project_id": project_id, "region": region, "job": job}
     )
-    job_id = result.reference.job_id
-    print("Submitted job ID {}.".format(job_id))
-    return job_id
+    response = operation.result()
 
+    # Dataproc job output is saved to the Cloud Storage bucket
+    # allocated to the job. Use regex to obtain the bucket and blob info.
```
Comment (loferris) [inline samples/snippets/submit_job_to_cluster.py:79]: I would use """ quotes instead of # for multi-line comments.

**Review by loferris — COMMENTED** [review]
Tentative LGTM, at least on format. I want to see it passing the sample tests first on pre-submit before approving.

**Comment by loferris** [comment]
I'm happy to approve once linting CI tests are passing!

**Comment by aman-ebay** [comment]
Moved these and other changes to PR 387

**Pull request closed without merging**